r2/r2/controllers/ipn.py

```diff
@@ -590,9 +590,9 @@ def finish(self, parameters, txn_id,
                         "supports the site and makes future development "
                         "possible. For example, one month of reddit gold "
                         "pays for 5 instance hours of reddit's servers.")
-            message += "\n\n" + strings.gold_benefits_msg
+            message += "\n\n" + _(strings.gold_benefits_msg)
```

Comment:
Yup, but we like to specify the variable name (like [here](https://github.com/reddit/reddit/pull/1472/files#diff-1c750c2158fff6e35fa4c6cb1aee45d7R123)) so that the placeholder is clearly spelled out in crowdin.

In crowdin, `_('%(number)s days until thanksgiving' % dict(number=2))` will let the translator know that there is a placeholder for `number` so that they can move that to the appropriate spot in the translation.